src/webgl/p5.RendererGL.js

```diff
@@ -2041,6 +2041,17 @@ p5.RendererGL = class RendererGL extends p5.Renderer {
   _setFillUniforms(fillShader) {
     fillShader.bindShader();
 
+    if (this._useMetalness > 0 && !this.curFillColor.every((value, index) =>
+      value === 1)) {
+      const metalnessFactor = Math.min(this._useMetalness / 100, 0.6);
```

Comment:
Did you have some visual examples that lead you to arrive at 0.6 as the max metalness and 0.4 as the min non-metalness instead of letting them both go from 0 to 1?